### PR TITLE
[RFR] Stakeholder group crud test

### PR DESCRIPTION
**Pull request by kpunwatk**
Signed-off-by: Karishma Punwatkar <kpunwatk@redhat.com>

	new file:   mta/tests/migration/controls/test_stakeholder_group.py
	modified:   pytest.ini

### CODE REVIEW

File: mta/tests/migration/controls/test_stakeholder_group.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_string
+
+
+@pytest.mark.sg
```
Comment (mguetta1) [inline mta/tests/migration/controls/test_stakeholder_group.py:7]: I think it is ok to mark it `stakeholder_groups`

File: mta/tests/migration/controls/test_stakeholder_group.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    # Assertion checking the created stakeholder group's name and description
+    check.equal(stakeholder_group_from_db.name, stakeholder_group.name)
+    check.equal(stakeholder_group_from_db.description, stakeholder_group.description)
+
+    # Edit the stakeholder group's name and description
+    stakeholder_group.name = generate_string(start="update name")
+    stakeholder_group.description = generate_string(start="update description")
+    stakeholdergroups_api.stakeholdergroups_id_put(stakeholder_group.id, stakeholder_group.to_dict())
+    update_stakeholder_group_from_db = stakeholdergroups_api.stakeholdergroups_id_get(stakeholder_group.id)
```
Comment (mguetta1) [inline mta/tests/migration/controls/test_stakeholder_group.py:20]: please rename to update**d**_stakeholder_group_from_db

File: pytest.ini [hunk 3]
```diff
@@ -6,3 +6,4 @@ markers =
     tags: marks tests as tags
     bs: marks tests as business services
     stakeholder: marks tests as stakeholder
+    sg: marks tests as stakeholder groups
```
Comment (mguetta1) [inline pytest.ini:9]: Please mark as stakeholder_groups

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: mta/tests/migration/controls/test_stakeholder_group.py (hunk 1 above)
Comment (kpunwatk) [reply mta/tests/migration/controls/test_stakeholder_group.py]: Done. Thanks!

File: mta/tests/migration/controls/test_stakeholder_group.py (hunk 2 above)
Comment (kpunwatk) [reply mta/tests/migration/controls/test_stakeholder_group.py]: Done

File: pytest.ini [hunk 4]
```diff
@@ -6,3 +6,4 @@ markers =
     tags: marks tests as tags
     bs: marks tests as business services
     stakeholder: marks tests as stakeholder
+    stakeholder_groups: marks tests as stakeholder stakeholder_groups
```
Comment (mguetta1) [inline pytest.ini:9]: @kpunwatk Please change this:
"stakeholder_groups: marks tests as stakeholder groups"

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: pytest.ini (hunk 4 above)
Comment (kpunwatk) [reply pytest.ini]: Done. Thanks for noticing

**Review by mguetta1 — APPROVED** [review]

**Review by sshveta — APPROVED** [review]

**Pull request merged**